src/main/kotlin/racingcar/RacingCarGame.kt

```diff
@@ -0,0 +1,31 @@
+package racingcar
+
+class RacingCarGame(private val carCount: Int, private val trialCount: Int) {
+    private val carList = mutableListOf<Car>()
+    val gameResult = mutableListOf<List<Int>>()
+
+    fun prepare() {
+        for (i in 0 until carCount) {
+            carList.add(Car())
+        }
+    }
+
+    fun play() {
+        for (c in 0 until trialCount) {
+            proceed()
+
+            record()
+        }
+    }
+
+    private fun record() {
+        val recordData = carList.map {it.position}
+        gameResult.add(recordData)
+    }
+
+    private fun proceed() {
+        carList.forEach {
+            it.proceed()
+        }
+    }
```

Comment:
```suggestion
    private fun proceed(): Unit = carList.forEach { it.proceed() }
```

해당 경우에는 문(statement) 보다 간결성을 위하여 식(expression)으로 변경하는 게 좋지 않을까요?